control-plane/connect-inject/controllers/pod/pod_controller_test.go

```diff
@@ -761,11 +763,1161 @@ func TestProxyConfigurationDelete(t *testing.T) {
+			//				IpPort: &pbmesh.IPPortAddress{
+			//					Port: uint32(1234),
+			//				},
+			//			},
+			//		},
+			//		{
+			//			DestinationRef: &pbresource.Reference{
+			//				Tenancy: &pbresource.Tenancy{
+			//					Partition: "",
+			//					Namespace: "",
+			//					PeerName:  "",
+			//				},
+			//				Name: "upstream2",
+			//			},
+			//			DestinationPort: "myPort2",
+			//			Datacenter:      "",
+			//			ListenAddr: &pbmesh.Upstream_IpPort{
+			//				IpPort: &pbmesh.IPPortAddress{
+			//					Port: uint32(2234),
+			//				},
+			//			},
+			//		},
+			//		{
+			//			DestinationRef: &pbresource.Reference{
+			//				Tenancy: &pbresource.Tenancy{
+			//					Partition: "",
+			//					Namespace: "ns1",
+			//					PeerName:  "",
+			//				},
+			//				Name: "upstream3",
+			//			},
+			//			DestinationPort: "myPort3",
+			//			Datacenter:      "",
+			//			ListenAddr: &pbmesh.Upstream_IpPort{
+			//				IpPort: &pbmesh.IPPortAddress{
+			//					Port: uint32(3234),
+			//				},
+			//			},
+			//		},
+			//		{
+			//			DestinationRef: &pbresource.Reference{
+			//				Tenancy: &pbresource.Tenancy{
+			//					Partition: "",
+			//					Namespace: "ns1",
+			//					PeerName:  "peer1",
+			//				},
+			//				Name: "upstream4",
+			//			},
+			//			DestinationPort: "myPort4",
+			//			Datacenter:      "",
+			//			ListenAddr: &pbmesh.Upstream_IpPort{
+			//				IpPort: &pbmesh.IPPortAddress{
+			//					Port: uint32(4234),
+			//				},
+			//			},
+			//		},
+			//	},
+			//},
+			consulNamespacesEnabled: true,
+			consulPartitionsEnabled: true,
+		},
+		{
+			name: "labeled when consul is unavailable, we don't return an error",
+			pod: func() *corev1.Pod {
+				pod1 := createPod(podName, "1.2.3.4", "foo", true, true)
+				pod1.Annotations[constants.AnnotationMeshDestinations] = "myPort.upstream1:1234:dc1"
+				return pod1
+			},
+			configEntry: func() api.ConfigEntry {
+				ce, _ := api.MakeConfigEntry(api.ProxyDefaults, "global")
+				pd := ce.(*api.ProxyConfigEntry)
+				pd.MeshGateway.Mode = "remote"
+				return pd
+			},
+			expErr: "upstream currently does not support datacenters: myPort.upstream1:1234:dc1",
+			// TODO: uncomment this and remove expErr when datacenters is supported
+			//expected: &pbmesh.Upstreams{
+			//	Workloads: &pbcatalog.WorkloadSelector{
+			//		Names: []string{podName},
+			//	},
+			//	Upstreams: []*pbmesh.Upstream{
+			//		{
+			//			DestinationRef: &pbresource.Reference{
+			//				Tenancy: &pbresource.Tenancy{
+			//					Partition: "",
+			//					Namespace: "",
+			//					PeerName:  "",
+			//				},
+			//				Name: "upstream1",
+			//			},
+			//			DestinationPort: "myPort",
+			//			Datacenter:      "dc1",
+			//			ListenAddr: &pbmesh.Upstream_IpPort{
+			//				IpPort: &pbmesh.IPPortAddress{
+			//					Port: uint32(1234),
+			//				},
+			//			},
+			//		},
+			//	},
+			//},
+			consulUnavailable:       true,
+			consulNamespacesEnabled: false,
+			consulPartitionsEnabled: false,
+		},
```

Comment:
I suspect this test case doesn't really function as expected for V2.

Reply:
I'll remove it